examples/oidc/css-api/nextjs/src/pages/api/integrations/[integrationID]/[environment]/roles.js

```diff
@@ -0,0 +1,40 @@
+import { fetchCssApiCredentials, apiErrorHandler } from '@/utils'
+
+async function handler(req, res) {
+    const { integrationID, environment } = req.query
+    const accessToken = await fetchCssApiCredentials();
+
+    // Get all roles for a given integration and Environment. 
+    // See https://api.loginproxy.gov.bc.ca/openapi/swagger#/Roles/get_integrations__integrationId___environment__roles for details.
+    if (req.method === 'GET') {
+        const roles = await fetch(`https://api-dev.loginproxy.gov.bc.ca/api/v1/integrations/${integrationID}/${environment}/roles`, {
```

Comment:
@jlangy lets not hard code the API URL but use an env var so that clients can either use sandbox or production API to test their integrations